src/test/java/org/mvel2/tests/core/res/PDFFieldUtil.java

```diff
@@ -5,9 +5,13 @@
 
 public class PDFFieldUtil {
   public int calculateAge(Date gebDat) {
-    long milliAge = System.currentTimeMillis() - gebDat.getTime();
+    Calendar c2 = Calendar.getInstance();
+    c2.set(2023,
+           11,
+           25); // 2023 dec 25
+    long milliAge = c2.getTimeInMillis() - gebDat.getTime();
```

Comment:
Previously, the test depends on `System.currentTimeMillis()` to calculate the age and starts to fail in 2024. Wow.